library/src/main/java/com/google/maps/android/data/Renderer.java

```diff
@@ -351,12 +388,77 @@ public Feature getContainerFeature(Object mapObject) {
     }
 
     /**
-     * Gets the images cache which is needed to download GroundOverlays and Marker Icon images
+     * Gets a cached image at the specified scale which is
+     * needed for Marker icon images
      *
-     * @return mImagesCache
+     * @param url URL to get cached image for
+     * @param scale scale to get image at
+     * @return scaled BitmapDescriptor
      */
-    public LruCache<String, Bitmap> getImagesCache() {
-        return mImagesCache;
+    public BitmapDescriptor getCachedMarkerImage(String url, double scale) {
```

Comment:
I'd expect that getting a cached image would be pretty fast just from this method name. However, since this method will also perform image scaling, which is time consuming, it would be good to add a note that this is the case.

Reply:
I added more details to the doc comment: https://github.com/googlemaps/android-maps-utils/pull/609/commits/f99a787285ef84fd0b57fc77cfd8e38f045c9055